distributions/kfdef/kfctl_openshift_v1.3.0.yaml

```diff
@@ -80,5 +80,5 @@ spec:
     name: kubeflow-apps
   repos:
   - name: manifests
-    uri: https://github.com/kubeflow/manifests/archive/v1.3-branch.tar.gz
-  version: v1.3-branch
+    uri: https://github.com/kubeflow/manifests/archive/v1.3.0.tar.gz
+  version: v1.3-tag
```

Comment:
Should this be `v1.3.0`?

Reply:
This is not really used anywhere, but fixed it